src/current/cockroachcloud/provision-a-cluster-with-terraform.md

```diff
@@ -179,6 +188,54 @@ The `terraform show` command shows detailed information of your cluster resource
 terraform show
 ~~~
 
+## Change a cluster's plan
+
+To change a CockroachDB {{ site.data.products.basic }} cluster's plan to CockroachDB {{ site.data.products.standard }} in place, or to change a CockroachDB {{ site.data.products.standard }} cluster to CockroachDB {{ site.data.products.basic }}, you can use Terraform.
+
+{{site.data.alerts.callout_info}}
+To migrate between CockroachDB {{ site.data.products.advanced }} and either CockroachDB {{ site.data.products.standard }} or CockroachDB {{ site.data.products.basic }}, you must create and configure a new cluster, back up the existing cluster's data, and restore the backup to the new cluster. Migration in place is not supported. Refer to [Customer-owned backups]({% link cockroachcloud/backup-and-restore-overview.md %}#customer-owned-backups).
```

Comment:
@mdlinville let's leave out Basic here - Advanced to Basic isn't a scenario we're planning to support, so customers will need to go Advanced to Standard, and then to Basic in the unlikely event that they would want to transition between Advanced and Basic.